mitosheet/mitosheet/saved_analyses/upgrade.py

```diff
@@ -358,6 +358,16 @@ def upgrade_saved_analysis_to_have_up_to_date_args(saved_analysis: Optional[Dict
     
     return saved_analysis
 
+def upgrade_saved_analysis_to_have_code(saved_analysis: Optional[Dict[str, Any]], code: Optional[str]) -> Optional[Dict[str, Any]]:
+
+    if saved_analysis is None:
+        return None
+    
+    if code is not None:
+        saved_analysis['code'] = code
+    
+    return saved_analysis
+
```

Comment:
This function doesn't actually add the code key when it doesn't exist. I think you need an else statement -- and we can just set it to empty for now? Also, can we make sure there's a test for this.